src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/DatabaseBackupTests.ps1

```diff
@@ -276,4 +276,82 @@ function Test-DatabaseGeoBackupPolicy
 	Set-AzureRmSqlDatabaseGeoBackupPolicy -ServerName $server.ServerName -ResourceGroupName $rg.ResourceGroupName -DatabaseName $db.DatabaseName -State Disabled
 	$result = Get-AzureRmSqlDatabaseGeoBackupPolicy -ServerName $server.ServerName -ResourceGroupName $rg.ResourceGroupName -DatabaseName $db.DatabaseName
 	Assert-True { $result.State -eq "Disabled" }
+}
+
+function Test-NewDatabaseRestorePoint
+{
+	# Setup
+	$location = "West US 2"
+	$serverVersion = "12.0";
+	$label = "label01";
+	$rg = Create-ResourceGroupForTest
+
+	try
+	{
+		$server = Create-ServerForTest $rg $location
+
+		# Create data warehouse database with all parameters.
+		$databaseName = Get-DatabaseName
+		$dwdb = New-AzureRmSqlDatabase -ResourceGroupName $rg.ResourceGroupName -ServerName $server.ServerName -DatabaseName $databaseName `
+			-Edition DataWarehouse -RequestedServiceObjectiveName DW100
+			
+		New-AzureRmSqlDatabaseRestorePoint -ResourceGroupName $rg.ResourceGroupName -ServerName $server.ServerName -DatabaseName $dwdb.DatabaseName -RestorePointLabel $label
+
+		# Get restore points from data warehouse database.
+		$restorePoints = Get-AzureRmSqlDatabaseRestorePoints -ResourceGroupName $rg.ResourceGroupName -ServerName $server.ServerName -DatabaseName $dwdb.DatabaseName
+
+		# We just created a restore point
+		Assert-AreEqual $restorePoints.Count 1
+		$restorePoint = $restorePoints[0]
+		Assert-AreEqual $restorePoint.RestorePointType DISCRETE
+		Assert-Null $restorePoint.EarliestRestoreDate
+		Assert-AreEqual $restorePoint.RestorePointCreationDate.Kind Utc
```

Comment:
Can you add an assert here for the label?

Reply:
added, good point